### PR TITLE
Crimapp 453 change maat schema unemployed journey

### PR DESCRIPTION
**Pull request by willmcb**
## Description of change

Adds minimal MAAT details for unemployed journey + fixture.

This adds minimal details required for unemployed and to test connections with MAAT - further work will be needed to add more means info in once this is built etc.

## Link to relevant ticket
https://dsdmoj.atlassian.net/browse/CRIMAPP-453

### CODE REVIEW

File: schemas/1.0/maat_means.json [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+                  "total_drawing",
+                  "total_profit",
+                  "director_salary",
+                  "share"
+                ]
+              }
+            }
+          },
+          "required":[
+            "paye"
```
Comment (arthurashman) [inline schemas/1.0/maat_means.json:213]: Why is paye required but not businesses?

**Review by arthurashman — APPROVED** [review]
Looks great. Tricky work, thank you for unpicking!

File: schemas/1.0/maat_means.json (hunk 1 above)
Comment (willmcb) [reply schemas/1.0/maat_means.json:213]: I am not sure, this is a subset of the overall means.json though and it is not a requirement in that schema either.

Comment (arthurashman) [reply schemas/1.0/maat_means.json:213]: Right, ok

File: schemas/1.0/maat_means.json [hunk 2]
```diff
@@ -0,0 +1,352 @@
+{
```
[the remark this answers is not in the document]
Comment (willmcb) [reply schemas/1.0/maat_means.json]: good idea have updated 👍

**Pull request merged**